AVPlayerExample/ViewController.m

```diff
@@ -277,8 +286,8 @@ - (void)doConnect {
         return;
     }
 
-    // Since we are configuring audio session explicitly, we will call setupAudioSession every time we attempt to connect.
-    [self setupAudioSession];
+    // Since we are configuring audio session explicitly, we will call StartAudioDevice every time we attempt to connect.
+    [self startAudioDevice];
```

Comment:
This example would be simpler if we just created the AudioDevice up front instead of starting / creating it at connect time. This may have been necessary with the old APIs, but it isn't now. WDYT @ptankTwilio?

Also, the audio device is enabled but we might want to leave it disabled until we get into the Room.